gtk4/src/auto/inscription.rs

```diff
@@ -0,0 +1,968 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files.git)
+// DO NOT EDIT
+
+use crate::Accessible;
+use crate::AccessibleRole;
+use crate::Align;
+use crate::Buildable;
+use crate::ConstraintTarget;
+use crate::LayoutManager;
+use crate::Overflow;
+use crate::Widget;
+use glib::object::Cast;
+use glib::object::IsA;
+#[cfg(any(feature = "v4_8", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+use glib::object::ObjectType as ObjectType_;
+#[cfg(any(feature = "v4_8", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+use glib::signal::connect_raw;
+#[cfg(any(feature = "v4_8", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+use glib::signal::SignalHandlerId;
+#[cfg(any(feature = "v4_8", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+use glib::translate::*;
+use glib::StaticType;
+use glib::ToValue;
+#[cfg(any(feature = "v4_8", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+use std::boxed::Box as Box_;
+use std::fmt;
+#[cfg(any(feature = "v4_8", feature = "dox"))]
+#[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+use std::mem::transmute;
+
+glib::wrapper! {
+    #[doc(alias = "GtkInscription")]
+    pub struct Inscription(Object<ffi::GtkInscription, ffi::GtkInscriptionClass>) @extends Widget, @implements Accessible, Buildable, ConstraintTarget;
+
+    match fn {
+        type_ => || ffi::gtk_inscription_get_type(),
+    }
+}
+
+impl Inscription {
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
```

Comment:
it is weird that all the methods are guarded with v4_8 but not the struct itself instead.